win32-deps/meson.build

```diff
@@ -1,24 +1,28 @@
-
 if sys_windows
 
   # Freetype
-  freetype_inc = include_directories('freetype.2.8.0.1/build/native/include/freetype')
-  freetype_lib = static_library('freetype',
-    include_directories: 'freetype.2.8.0.1/build/native/lib/x64/v141/static/Release'
+  freetype2_inc_dir = include_directories('freetype.2.8.0.1/build/native/include')
+  freetype2_lib_dir = 'freetype.2.8.0.1/build/native/lib/x64/v141/static/Release'
+  freetype2_lib = cc.find_library('freetype28',
+    dirs: meson.build_root() + '/../win32-deps/' + freetype2_lib_dir,
+    required: true,
   )
-  freetype = declare_dependency(
-    link_with: freetype_lib,
-    include_directories: freetype_inc
+  freetype2 = declare_dependency(
+    dependencies: freetype2_lib,
+    include_directories: freetype2_inc_dir,
   )
```

Comment:
I have a feeling this is wrong. Mind to explain the idea?

Reply:
To be honest this looks _a lot_ like what should be a wrap-dependency. I think we should change this into [freetype2 wrap dependency](https://wrapdb.mesonbuild.com/freetype2).

Reply:
hmm, this is almost that, except that it is not intended to be build by meson. Just to mention I followed some meson manual about linking with your shipped binaries or something like that.